pkgbuild/create-installer-mac.sh

```diff
@@ -25,7 +25,7 @@ SIGN_CERT=
 SIGN_CMD=
 if [ ! -z "$CERTIFICATE" ]; then
   SIGN_CMD="--sign"
-  SIGN_CERT="${CERTIFICATE}"
+  SIGN_CERT="'${CERTIFICATE}'"
```

Comment:
I think it reads better without the negation, assigning both variables exactly once:
```
if [ -z "$CERTIFICATE" ] ; then
  SIGN_CMD=
  SIGN_CERT=
else
  SIGN_CMD="--sign"
  SIGN_CERT="'${CERTIFICATE}'"
fi
```

Reply:
Good idea. It is safer to set something to blank rather than assume it is not set.